### PR TITLE
Allow collaborators to self-assign issues

### PR DESCRIPTION
**Pull request by albertvillanova**
Allow collaborators (without write access to the repository) to self-assign issues.

In order to self-assign an issue, they have to comment it with the word: `#take` or `#self-assign`.

### CODE REVIEW

**Comment by albertvillanova** [comment]
What do you think, @lhoestq? 😉 

I think this is another step to facilitate community contributions.

**Review by lhoestq — APPROVED** [review]
Good idea ! Is there something similar in `transformers` ?
Also it would be good to mention this in [CONTRIBUTING.md](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md)

**Comment by albertvillanova** [comment]
@lhoestq, I don't know if it exists in `transformers`... I picked the idea from `scikit-learn`, where I have previously collaborated...

And sure, this must be documented! I just wanted first to know your opinion...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ Many thanks in advance to every contributor.
...
+
+Some of them may have the label `help wanted`: that means that any contributor is welcomed!
+
+If you would like to work on any of the open Issues:
+
+1. Make sure it is not already assigned to someone else. You have the assignee (if any) on the top of the right column of the Issue page.
+
+1. You can self-assign it by commenting on the Issue page with one of the keywords: `#take` or `#self-assign`.
+
+1. Work on your self-assigned issue and eventually create a Pull Request.
```
Comment (lhoestq) [inline CONTRIBUTING.md:25]:
```suggestion
2. You can self-assign it by commenting on the Issue page with one of the keywords: `#take` or `#self-assign`.

3. Work on your self-assigned issue and eventually create a Pull Request.

```

**Review by lhoestq — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (albertvillanova) [reply CONTRIBUTING.md:25]: Numbering of ordered lists is made automatically. The markdown symbol for each item in an ordered list is `1. `: https://guides.github.com/features/mastering-markdown/#syntax
😉

**Pull request merged**